lambdas/functions/control-plane/src/aws/runners.ts

```diff
@@ -46,6 +48,9 @@ function constructFilters(filters?: Runners.ListRunnerFilters): Ec2Filter[][] {
       ec2FiltersBase.push({ Name: `tag:ghr:Type`, Values: [filters.runnerType] });
       ec2FiltersBase.push({ Name: `tag:ghr:Owner`, Values: [filters.runnerOwner] });
     }
+    if (filters.orphan) {
+      ec2FiltersBase.push({ Name: 'tag:ghr:orphan', Values: ['true'] });
```

Comment:
```suggestion
      ec2FiltersBase.push({ Name: 'tag:ghr:Orphan', Values: ['true'] });
```
Everything else uses caps?

Reply:
Most of the tags we use are in lower case like `ghr:environment`, `ghr:ssm_config_path` would suggest we stick to lower case an update owner and type to lower as well in a sperate PR.